model/telemetry/metrics.yaml

```diff
@@ -0,0 +1,76 @@
+groups:
+  - id: metric.telemetry.sdk.trace.spans.ended
+    type: metric
+    metric_name: telemetry.sdk.trace.spans.ended
+    stability: experimental
+    brief: "The number of spans which have been ended"
+    instrument: counter
+    unit: "1"
+
+  - id: metric.telemetry.sdk.trace.spans.sampled
+    type: metric
+    metric_name: telemetry.sdk.trace.spans.sampled
+    stability: experimental
+    brief: "The number of spans which have been ended AND are sampled"
+    instrument: counter
+    unit: "1"
+
+  - id: metric.telemetry.sdk.trace.processor.queue_size
+    type: metric
+    metric_name: telemetry.sdk.trace.processor.queue_size
```

Comment:
do this kind of queues also have a limit? If so should we maybe introduce that as an additional metric or so?

Reply:
Yes, makes sense. This allows the detection of a full queue, which is the most relevant use case for the `queue_size` metric anyway IMO.

Reply:
Done in c62be64